CHANGELOG.md

```diff
@@ -4,6 +4,61 @@ This document includes a curated changelog for each release. We also publish a c
 a [GitHub release](https://github.com/nginxinc/nginx-gateway-fabric/releases), which, by contrast, is auto-generated
 and includes links to all PRs that went into the release.
 
+## Release 1.5.0
+
+_November 20, 2024_
+
+BREAKING CHANGES:
+
+- NGINX Plus R33 support added. This release of NGINX Plus requires a valid JSON Web Token (JWT) in order to run. Users of NGINX Plus _must_ have this JWT added to a Secret before installing NGINX Gateway Fabric v1.5.0. See the [NGINX Plus JWT](https://docs.nginx.com/nginx-gateway-fabric/installation/nginx-plus-jwt/) guide for information on setting this up. [2760](https://github.com/nginxinc/nginx-gateway-fabric/pull/2760)
+
+FEATURES:
+
+- Add support to retain client IP information. [2284](https://github.com/nginxinc/nginx-gateway-fabric/pull/2284)
+- Add ability to configure data plane error log level. [2603](https://github.com/nginxinc/nginx-gateway-fabric/pull/2603)
+- Introduce SnippetsFilter API, which allows users to inject custom NGINX configuration via an HTTPRoute or GRPCRoute filter. See the [SnippetsFilter guide](https://docs.nginx.com/nginx-gateway-fabric/how-to/traffic-management/snippets/) for information on how to use SnippetsFilters. [2604](https://github.com/nginxinc/nginx-gateway-fabric/pull/2604)
```

Comment:
Yeah agreed! Past tense is better

Reply:
@salonichf5 can we wait for @ADubhlaoich to chime in on the tense? Looking through our old release notes and we use a mix of tenses, and I'd like to be consistent moving forward.

Reply:
okay, sounds good. I'll update as needed. I followed the process for previous release which was sort of mix of present tense or past tense.

Reply:
Copying and pasting this in the multiple threads it's come up in (Just in case it's lost):

We stick to present tense and imperative mood: which I would think of as "a command to do something now".

If it helps, the mental model is as though you are giving a computer an instruction.

This style is used in conventional commits, as though you are instructing git to make changes while going through revisions.

- "Add new feature"
- "Change thing"
- "Remove item"

And so forth. I hadn't noticed it was inconsistent in prior release: my apologies.

Reply:
Sounds good. I'll update it!